File: tests/fixtures/sample_buggy.py
import asyncio


class UserService:
    def get_user_sync(self, user_id: int) -> dict:
        return {"id": user_id, "name": "Ada"}


async def get_user(user_id: int):
    service = UserService()
    user_data = await service.get_user_sync(user_id)
    return user_data


def top_level_helper():
    x = 1
    y = 2
    return x + y


if __name__ == "__main__":
    asyncio.run(get_user(1))

File: tests/fixtures/sample_buggy.ts
type User = { id: number; name: string };

class UserService {
  getUserSync(userId: number): User {
    return { id: userId, name: "Ada" };
  }
}

async function getUser(userId: number): Promise<User> {
  const service = new UserService();
  const userData: User = await service.getUserSync(userId);
  return userData;
}

getUser(1);
